Dockerfile

```diff
@@ -1,9 +1,8 @@
 FROM registry.ci.openshift.org/ocp/4.11:base
-
 ARG KERNEL_VERSION=''
 ARG RT_KERNEL_VERSION=''
 ARG RHEL_VERSION=''
-
+ARG RT_ENABLED
```

Comment:
I don't see this variable being used elsewhere.

Reply:
Outdated variable. Not being used anymore.
Thank you.